extensions/amp-subscriptions/0.1/local-subscription-platform-remote.js

```diff
@@ -83,26 +83,9 @@ export class LocalSubscriptionRemotePlatform extends LocalSubscriptionBasePlatfo
     return !!this.pingbackUrl_;
   }
 
-  /**
-   * Format data for pingback
-   * @param {./entitlement.Entitlement|Array<./entitlement.Entitlement>} entitlements
-   * @return {string}
-   * @private
-   */
-  stringifyPingbackData_(entitlements) {
-    if (isArray(entitlements)) {
-      const entitlementArray = [];
-      entitlements.forEach((ent) => {
-        entitlementArray.push(ent.jsonForPingback());
-      });
-      return JSON.stringify(entitlementArray);
-    }
-    return JSON.stringify(entitlements.jsonForPingback());
-  }
-
   /** @override */
   pingback(selectedEntitlement) {
-    if (!this.isPingbackEnabled) {
+    if (!this.isPingbackEnabled() && !this.isDeferredAccountCreationEnabled()) {
```

Comment:
This might be a bit problematic. If user refuses account creation, I think publisher should continue receiving pingbacks - it's their impression and I think this is their data to receive.

Reply:
+1